libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -260,11 +260,14 @@ void AP_MotorsHeli_Dual::set_desired_rotor_speed(float desired_speed)
 // calculate_armed_scalars
 void AP_MotorsHeli_Dual::calculate_armed_scalars()
 {
+    float thrcrv[5];
+    int i;
+    for (i = 0; i < 5; i++) {thrcrv[i]=_rsc_thrcrv[i]*0.001f;} 
```

Comment:
minor style thing, format like this:
for (uint8_t i=0; i<5; i++) {
    thrcrv[i]=_rsc_thrcrv[i]*0.001f;
}